backend/copr_backend/helpers.py

```diff
@@ -627,7 +628,40 @@ def get_chroot_arch(chroot):
     return chroot.rsplit("-", 2)[2]
 
 
-def pkg_name_evr(srpm_path, logger):
+def find_spec_file(folder):
+    """
+    Returns spec file path or None
+    """
+    spec_pattern = os.path.join(folder, "*.spec")
+    result = glob.glob(spec_pattern)
+    if not result:
+        return None
+
+    return result[0]
+
+
+def format_evr(epoch, version, release):
+    """
+    Return evr in format (epoch:)version-release
+    """
+    if epoch is not None and epoch.isdigit():
+        return f"{epoch}:{version}-{release}"
+
+    return f"{version}-{release}"
+
+
+def pkg_name_evr(spec_file_path, srpm_path, logger):
+    """
+    Queries a SRPM or spec file for its name and evr (epoch:version-release)
+    """
```

Comment:
I'd wrap the whole call of this method into a `try-except` block;  eventually raising a clear human-readable error to the user if something goes wrong.